spec/dummy/log/development.log

```diff
@@ -0,0 +1 @@
+DEPRECATION WARNING: ActiveSupport.halt_callback_chains_on_return_false= is deprecated and will be removed in Rails 5.2. (called from <top (required)> at /Users/jsharpe/rails_critical_css_server/spec/dummy/config/initializers/new_framework_defaults.rb:21)
```

Comment:
Did you mean to commit this file? I'm not sure what this is, but it looks like it should be removed and/or .gitignored?